### PR TITLE
Create ClosedOrdersProcessor and test it

### PR DESCRIPTION
**Pull request by arn3tt**

### CODE REVIEW

File: src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                }
+            } else {
+                LOGGER.info(String.format("Order %s has no instance associated", order.getId()));
+            }
+        }
+    }
+
+    private void deactivateOrder(Order order) {
+        Map<String, Order> activeOrdersMap = sharedOrderHolders.getActiveOrdersMap();
+        if (activeOrdersMap.get(order.getId()) != null) {
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java]: Is the containsKey method from Map more apropriated?

File: src/test/java/org/fogbowcloud/manager/core/threads/TestClosedComputeOrdersProcessor.java [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+    public void testProcessClosedRemoteOrder() {
+
+    }
+
+    @Test
+    public void testProcessClosedOrderFails() {
+
+    }
+
+//    /**
```
Comment (paulofelipefeitosa) [inline src/test/java/org/fogbowcloud/manager/core/threads/TestClosedComputeOrdersProcessor.java]: Why theses tests are commented?

File: src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    @Override
+    public void run() {
+        boolean isActive = true;
+        while (isActive) {
+            try {
+                Order order = this.closedOrders.getNext();
+                if (order != null) {
+                    try {
+                        processClosedOrder(order);
+                        closedOrders.removeItem(order);
```
Comment (paulofelipefeitosa) [inline src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java]: Why you are removing the order from closed list in the run method?

File: src/test/java/org/fogbowcloud/manager/core/threads/TestClosedComputeOrdersProcessor.java (hunk 2 above)
Comment (arn3tt) [reply src/test/java/org/fogbowcloud/manager/core/threads/TestClosedComputeOrdersProcessor.java]: I forgot to remove them, will fix this.

File: src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java (hunk 3 above)
Comment (arn3tt) [reply src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java]: Please take a look at the last page of our Design Contract. The pseudo-code of this member is detailed there.

File: src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java (hunk 1 above)
Comment (arn3tt) [reply src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java]: Nice catch! Will fix this in the next commits.

**Comment by arn3tt** [comment]
@paulofelipefeitosa can you review this PR again? I fixed and discussed what you found.

File: src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java (hunk 3 above)
Comment (paulofelipefeitosa) [reply src/main/java/org/fogbowcloud/manager/core/threads/ClosedComputeOrdersProcessor.java]: Sorry, you do not understand my question, I know that is necessary to remove the order from the closed orders list, I am questioning why you are doing it at the run method? Is more appropriated to do it at the process method?

**Pull request merged**